src/Xamarin.Android.Build.Tasks/Tasks/JavaDoc.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Linq;
+using System.IO;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System.Text;
+using System.Collections.Generic;
+using Xamarin.Android.Tools;
+using Xamarin.Android.Tools.Aidl;
+
+namespace Xamarin.Android.Tasks
+{
+
+	public class JavaDoc : JavaToolTask
+	{
+		public string [] SourceDirectories { get; set; }
+
+		[Output]
```

Comment:
The `[Output]` custom attribute is for use with the `<Output/>` element, which isn't being used. `[Output]` isn't needed.